README.md

```diff
@@ -82,63 +82,113 @@ The project will automatically be deployed to github-pages.
+Now we can redirect this link:
 
-The website is currently served from a `/new` directory on the production server. This can be achieved by running the following command to create a symlink: `sudo ln -sT /home/rits/oracc-search-front-end/dist/oracc /home/oracc/www/new`
+```sh
+rits@build-oracc:~$ sudo ln -sfT /home/rits/www/oracc-search-front-end/1.2.3 /home/oracc/www/new
+```
 
-This will symlink each file and folder to the new directory. You can check that the symlink has been created by running: `ls -la ./ | grep "\->"`
+In case of trouble, we can roll back to the old version using the same command with the old version number we found out earlier:
 
-Note that if you are getting a `403 forbidden` error from Apache you probably need to set the correct folder privileges. Make sure that the `/home/rits` folder and all its child folders relating to Angular have at least `drwxr-xr-x` privileges. You can set these privileges with: `sudo chmod 755 /home/rits` and you can check current privileges with: `ls -l /home/rits` .
+```sh
+rits@build-oracc:~$ sudo ln -sfT /home/rits/www/oracc-search-front-end/1.2.2 /home/oracc/www/new
+```
 
-## Further Apache configurations
+The production server will instantly begin serving this
+new version. If you like, you can delete old, obsolete
+versions like so:
+
+```sh
+rits@build-oracc:~$ rm /home/rits/www/oracc-search-front-end/1.1.0/ -rf
+```
+
+Note that if you are getting a `403 forbidden` error from Apache you probably need to set the correct folder privileges. Make sure that the `/home/rits/www` folder and all its child folders relating to Angular have at least `drwxr-xr-x` privileges. You can set these privileges with: `sudo chmod 755 /home/rits/www` and you can check current privileges with: `ls -l /home/rits/www` .
+
+### Further Apache configurations
 
 The Oracc server runs on Ubuntu and exposes the Oracc website via an Apache web server. Therefore, you may need to configure Apache to appropriately serve the static content generated via the Angular build process.
 
-Specifically, you will need to add a rewrite rule so that the angular app pages fall back to `/new/index.html`. This has currenty been achieved by adding the following rewrite rules to `/etc/apache2/sites-enabled/oracc-vhost-ssl.conf` :
+The following rules need to exist in the file `/etc/apache2/sites-enabled/oracc-vhost-ssl.conf`:
+
+1. The angular app should be served at `/new`
+2. The oracc-rest API should be served at `/oracc-rest-api/`
+
+These rules look like this:
 
 ```apacheconf
 <VirtualHost *:443>
 ...
 
-# Angular website config - rewrites routes back to /new/index.html
-RewriteCond %{REQUEST_FILENAME} "^/new(/.*)?$"
-RewriteCond %{DOCUMENT_ROOT}%{REQUEST_URI} -f [OR]
-RewriteCond %{DOCUMENT_ROOT}%{REQUEST_URI} -d
-RewriteRule ^ - [L]
-RewriteCond %{REQUEST_FILENAME} "^/new(/.*)?$"
-RewriteRule ^(.+)$ /new/index.html [L]
+    <Location /oracc-rest-api>
+        ProxyPass http://localhost:5001
+    </Location>
```

Comment:
Does this require any additional modules to be activated on Apache? If so, we should probably document them.

Reply:
Also, does this mean that the backend docker needs to be exposed on localhost:5001? Is this configured in your latest backend PR?

Reply:
Excellent point, it requires mod_proxy and mod_proxy_http.
Yes, backend needs to be listening on 5001, but it doesn't need to be exposed to the outside world.